modules/vaos/spec/services/middleware/vaos_logging_integration_spec.rb

```diff
@@ -45,12 +49,116 @@
       end
     end
 
+    context 'with a timeout' do
+      around do |example|
+        VCR.turn_off!
+        WebMock.allow_net_connect!
+        example.run
+        VCR.turn_on!
+        WebMock.disable_net_connect!
+      end
```

Comment:
I'm putting my Kam hat on =).
Nitpick, this can probably be broken out into a method.